src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -389,6 +389,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
 #End Region
 
 #Region "Buffer helpers"
+
+        Private Function NextAre(offset As Integer, chars As String) As Boolean
+            Debug.Assert(Not String.IsNullOrEmpty(chars))
+            Dim n = chars.Length
+            Dim i = -1
+            If CanGetCharAtOffset(offset + n) Then
+                Do
+                    i += 1
+                Loop While i < n AndAlso chars(i) = PeekAheadChar(offset + i)
+            End If
+            Return i = n
+        End Function
```

Comment:
So noted and added in [6396a55](https://github.com/dotnet/roslyn/commit/6396a558fec11b08b2f9a752a9cd92744811d3b6)